contracts/dapis/proxies/ProxyFactory.sol

```diff
@@ -111,4 +112,97 @@ contract ProxyFactory is IProxyFactory {
             metadata
         );
     }
+
+    /// @notice Computes the address of the data feed proxy
+    /// @param dataFeedId Data feed ID
+    /// @param metadata Metadata associated with the proxy
```

Comment:
maybe we can be a little more specific what metadata is in the @dev section

Reply:
I think we can't be more specific than "metadata" because this can be used for all kinds of stuff ranging from coverage to OEV to functionality that third parties implement